shared/yarn.lock

```diff
@@ -4638,13 +4638,13 @@ debug@2.2.0:
   dependencies:
     ms "0.7.1"
 
-debug@2.6.8, debug@^2.1.1, debug@^2.1.3, debug@^2.2.0, debug@^2.6.6, debug@^2.6.8:
+debug@2.6.8, debug@^2.1.3, debug@^2.2.0, debug@^2.6.6, debug@^2.6.8:
```

Comment:
you didn't intend to change `yarn.lock`, right?

Reply:
nope! I'll revert that